client/landing/stepper/hooks/use-site-has-wpcom-migration-plugin.ts

```diff
@@ -19,7 +19,9 @@ export const useSiteHasWPCOMMigrationPlugin = ( siteId: number, options: Options
 		enabled,
 	} );
 
-	const pluginRecord = data?.plugins.find( ( plugin: any ) => plugin.slug === 'wpcom-migration' );
+	const pluginRecord = ( data as any )?.plugins.find(
+		( plugin: any ) => plugin.slug === 'wpcom-migration'
+	);
```

Comment:
This adds an `any` cast, but it was basically already there. `useQuery()` was returning an `any` before, and with the new typings this just has to be explicit.